client/src/components/UsedOnTable/UsedOnTable.js

```diff
@@ -64,23 +63,24 @@ class UsedOnTable extends PureComponent {
       id,
       title,
       type,
-      state,
       link,
     } = data;
 
-    const badge = (state)
-      ? <span className={classnames('badge', 'used-on__badge', `status-${state}`)}>{state}</span>
-      : null;
-
     const titleLabel = (link)
-      ? <a className="used-on__edit-link" href={link}>{title} {badge}</a>
-      : <span>{title} {badge}</span>;
+      ? <a className="used-on__edit-link" href={link}>{title}</a>
+      : <span>{title}</span>;
+
+    const handleTrClick = () => {
+      window.location = link;
+    };
 
     return (
-      <tr key={id}>
+      <tr key={id} className="used-on__row" onClick={handleTrClick} role="button">
```

Comment:
This might cause some accessibility problems. I don't think you can tab to a `<tr>` element.

Reply:
You can if you tell the browser it's a button. It may need a tabindex prop too though. Ideally we avoid doing this if possible, but GridField based UIs are pretty table dominated at the moment.

Reply:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role#Basic_button_example